Flow.Launcher.Test/FuzzyMatcherTest.cs

```diff
@@ -129,11 +131,11 @@ public void GivenQueryString_WhenAppliedPrecisionFiltering_ThenShouldReturnGreat
 
         [TestCase(Chrome, Chrome, 157)]
         [TestCase(Chrome, LastIsChrome, 147)]
-        [TestCase(Chrome, HelpCureHopeRaiseOnMindEntityChrome, 25)]
+        [TestCase(Chrome, HelpCureHopeRaiseOnMindEntityChrome, 90)]
```

Comment:
this shouldnt have such a high score that it shows, acronym search matches it despite 'Help' at the start?

Reply:
hmm despite the penalty, i wonder if this is will bring up irrelevant results

Reply:
I am not quite sure, but there shouldn't be few strings that will act like that. Maybe we can release a pre-release and check how people respond to it.